submissions/burning-questions/burning-questions.md

```diff
@@ -0,0 +1,367 @@
+
+`Total processing energy (E) = Machine power usage (P) * time of 1 crafting cycle (t_c) * numnber of crafting cycles (n) / crafting speed (C)`
+
+`E = P * t_c * n / C`
+
+Here is what the modules do:
+
+- **Efficiency modules:** They reduce the power usage of the fuel processing machines ("P"). Their overall effects on energy gains are small but noteworthy for most recipes. I would definitely recommend using efficiency 1 modules in all of the fuel processing machines if you don't use any other modules because they are quite cheap to make and they cut down power usage by 60% or even 80% (the efficiency limit), depending on how many module slots the machines have.
+
+- **Speed modules:** One effect of speed modules is increasing power usage ("P") just like efficiency modules, but in the opposite direction, hence reducing our energy gains. The other effect of speed modules is increasing the crafting speeds of machines ("C"), which as seen from the equation, reduces the power usage. For every speed module, the increase in P is greater than the increase in C, which means that the overall effect of any speed module is an increase in E, which results in decreased energy gains overall. However, speed modules can be used to mitigate the slowdown from productivity modules, which might provide energy gains overall.
+
+- **Productivity modules:** They have the most interesting effects. They increase power usage ("P") and decrease crafting speed ("C"), both of which lead to an increase in E, which decreases the overall energy gains. However, productivity modules also increase the amount of processed fuel produced. Furthermore, if we keep using them in every step of the processing, we can get more and more fuel across the system! The energy gains from the extra fuel might outweigh all the extra power consumption.
+
+
+
+{% include image.html
+
+src= 'src'
+ alt='Module effects on power consumption'
+ caption='Module effects on power consumption'
+
+%}
+
+Now let's demonstrate the use of modules. First, let's look at the case of efficiency in coal liquification with heavy oil cracking. Refineries and chemical plants have 3 module slots, so we can place 3 efficiency 1 modules to reduce power consumptions by 80% for each: 
+
+{% include image.html
+
+src= 'src'
+ alt='Processing coal into solid fuel with heavy oil cracking, at minimum power consumption'
+ caption='Processing coal into solid fuel with heavy oil cracking, at minimum power consumption'
+
+%}
+
+The total power consumption drops from about 3.28MJ to about 0.66MJ, which translates to the energy gains of 116%, compared to the default case of 103%. This increase is nice but it is not that large because processing energy was already relatively small. Now let us look at the same example with 3 productivity 3 modules instead to see the effects of maximum productivity:
```

Comment:
```suggestion
The total power consumption drops from about 3.28MJ to about 0.66MJ, which translates to the energy gains of 116%, compared to the default case of 103%. This increase is nice, but it is not that significant since the processing energy cost was already relatively small. Now, let us look at the same example with 3 productivity 3 modules instead to see the effects of maximum productivity:
```